deploy/operator/setup_lso.sh

```diff
@@ -34,6 +34,32 @@ function install_lso() {
       "${index_image}" "${LOCAL_REGISTRY}" "${AUTHFILE}" "${catalog_source_name}"
   fi
 
+  OC_VERSION_MAJOR_MINOR=$(oc version -o json | jq --raw-output '.openshiftVersion' | cut -d'.' -f1-2)
+  if [[ ${OC_VERSION_MAJOR_MINOR} == "4.9" ]]; then
+      # LSO has not been published to the 4.9 redhat-operators catalog, so
+      # it cannot be installed on OpenShift 4.9. Until this is resolved,
+      # we explicitly install the 4.8 catalog as redhat-operators-v48
+      # and then subscribe to the LSO version from the 4.8 rather than the 4.9 catalog.
+      # TODO: Remove this once LSO is published to the 4.9 catalog.
```

Comment:
```suggestion
      # TODO: Revert https://github.com/openshift/assisted-service/pull/2688 once LSO is published to the 4.9 catalog.
```

Reply:
I'm against tying up code / commit messages to GitHub URLs, because it makes it impossible to migrate from GitHub in the future (migrating git repos otherwise is super easy, literally just git clone and git push)

If anyone needs to revert this change in the future, `git blame` then `git revert <commit_hash>` is easy enough

Reply:
Not sure it's worth being that generic
I'm all for a more practical usage of the tools we currently have

Reply:
@YuviGold what do you think?

Reply:
> I'm all for a more practical usage of the tools we currently have

That's a fair point, it's more of a principal for me. 

Also in this case I don't think my alternative is too different -

- GitHub forces their PR IDs into your merge commit messages anyway, so it's easy to find the PR even without the comment you suggest, if you just do `git blame`

- I don't see how `git blame` and `git revert <commit_hash>` are harder than using the GitHub UI to do the revert (especially with GitHub CLI's to quickly create PRs from the terminal)

Reply:
Not a big deal in my opinion, but I agree I would expect a contributor to be familiar of `git blame` or other helpful tools.

AFAIK Prow adds the PR number to the merge commit (depends on the tide merge strategy for the repository)